### DISCUSSION TITLE
Mozilla rejects listing Pulsar as an alternative to VS Code.

### DISCUSSION THREAD

**Discussion by trusktr** [pulsar-edit/.github#246] [Hacking Pulsar]
Take a look at this PR:

- https://github.com/mdn/content/pull/29570

**Comment by confused-Techie** [comment]
Thanks for the heads up! And thanks for submitting that original PR.

I think your ask for clarification is extremely valid, even if it's never received any response.

The only thing I can think of that adding our link does to break their policy:

> Generic or non-specific (e.g., a vendor's home page, instead of the related documentation)

But for the above rule, this is exactly what they do linking to Visual Studio, so who knows.

Or they otherwise worry the project won't be around for long enough to be worth listing.

But what's really interesting is just a few paragraphs down the page, they list a slew of other third party editors and software, including editors like:
* Notepad++
* Sublime Text
* GNU Emacs
* VIM
* Dreamweaver
* WebStorm

Which this list up until Aug 30, 2022 listed [Atom](https://github.com/mdn/content/commit/e3cc9163bf6e88054ab2100379b5eb2a6d3fc846).

So if I had to hazard a guess, every other piece of software listed is well known, popular, or backed by a large company that's unlikely to fail. To me it seems entirely likely that the maintainer that closed your PR had never heard of us, and based solely on that decided to close the PR.

Disappointing for sure, but hopefully is the sort of problem that will be sorted out naturally. That if the editor gains a big enough following would be something that the next time the same PR is made would then be accepted. But is rather disappointing to see them take such a blasé approach to your PRs review, but will have to assume it's because the majority of the time they get this exact same PR is for spam purposes.
